jobs/v3/api_client/auto_complete_sample_test.py

```diff
@@ -28,7 +28,7 @@ def company_name():
 
 
 def test_auto_complete_sample(company_name, capsys):
-    @eventually_consistent.call
+    @eventually_consistent.call(tries=5)
```

Comment:
Can we switch to `flaky` or `retry` over `eventually_consistent `?

Reply:
@kurtisvg Thanks for the comment. To be honest, we have different opinions.

I think eventually_consistent has some benefit. It gives semantics.

It marks the tests as eventually consistent, which means it is a correct thing to retry.
I quite don't like to use `flaky` or `retry` for eventually consistent tests, because we can not distinguish eventually consistent tests from usual flakes.

Also, it is a great shortcut for eventually consistent issues with exponential back off. There's no need for defining helper callback.

So, for eventually consistent tests, I strongly *want* to stick with `eventually_consistent`.

Reply:
So, if we ultimately decide to eliminate gcp-devrel-py-tools, I'll replace `eventually_consistent` with `retrying`. However, I'll stick to eventally_consistent in this PR.

Can you file an issue for eliminating gcp-devrel-py-tools so that we can discuss further there?